unity-environment/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -253,6 +337,15 @@ void InitializeEnvironment()
         ConfigureEnvironment();
     }
 
+    void HandleLog(string logString, string stackTrace, LogType type)
```

Comment:
Why are we not using the standard logging infrastructure (https://docs.unity3d.com/ScriptReference/Logger.Log.html) ?

Reply:
I don't understand. I do not want to log in the unity console, I want that every time something is logged in the console, it is written somewhere else as well.

Reply:
Sure, that's why you have the https://docs.unity3d.com/ScriptReference/ILogHandler.html

The less new mechanisms you invent, the easier it will be to maintain the code.